src/commands/Animal/catfact.ts

```diff
@@ -2,29 +2,28 @@ import { LanguageKeys } from '#lib/i18n/languageKeys';
 import { SkyraCommand } from '#lib/structures';
 import { assetsFolder } from '#utils/constants';
 import { ApplyOptions } from '@sapphire/decorators';
+import { send } from '@skyra/editable-commands';
 import { Message, MessageEmbed } from 'discord.js';
 import { readFile } from 'fs/promises';
 import { join } from 'path';
 
 @ApplyOptions<SkyraCommand.Options>({
 	aliases: ['kittenfact'],
-	cooldown: 10,
```

Comment:
intentionally removed?

Reply:
Yeah, catfact uses a local json as opposed to a remote API. So there is no point in a cooldown.